### PR TITLE
Added highlight section to change log for _field_caps enhancement

### PR DESCRIPTION
**Pull request by piergm**
Relates to: [#103651](https://github.com/elastic/elasticsearch/pull/103651)

### CODE REVIEW

File: docs/changelog/103651.yaml [hunk 1]
```diff
@@ -3,3 +3,14 @@ summary: Flag in `_field_caps` to return only fields with values in index
 area: Search
 type: enhancement
 issues: []
+highlight:
+  title: Flag in `_field_caps` to return only fields with values in index
+  body: |-
+    We are adding a query parameter to the `_field_caps` APIs to allow filtering
```
Comment (javanna) [inline docs/changelog/103651.yaml:9]: I would rephrase the first part to something like: 

We added support for filtering the field capabilities API output by removing fields that don't have a value. This can be done through the newly added `include_empty_fields` parameter, which defaults to true.

I am kinda wondering if details about Kibana should be in the ES release notes. But no strong opinion on it.

Comment (piergm) [reply docs/changelog/103651.yaml:9]: Removing the Kibana section. After reading it again it feels out of place here.

**Review by javanna — APPROVED** [review]
